tests/Ninja/Loader/include.ninja

```diff
@@ -24,7 +24,7 @@ build b2: testrule input ${input2_var} xx${should_be_empty}xx
 
 # Check that we diagnose broken includes.
 #
-# CHECK-ERR: include.ninja:[[@LINE+1]]:8: error: unable to open input
+# CHECK-ERR: include.ninja:[[@LINE+1]]:8: error: unable to read input
```

Comment:
Is the exact error important here? The open/read aren't separate any more (just uses MemoryBuffer::getFile) and "read" seemed the more generic of the two.

Reply:
“read” seems reasonable to me.